content/blog/kubernetes-psp-deprecated.fr.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: Kubernetes va déprécier les PodSecurityPolicy
+Date: 2021-01-20
+Category: Kubernetes
+Summary: "Kubernetes va déprécier, à partir de la `1.21`, les PSP, ces
+composantes permettant d'assurer la sécurité au niveau du pod. Pourquoi ? Quelles
+alternatives ?"
+Author: Romain Guichard
+image: images/thumbnails/kubernetes.png
+imgSocialNetwork: images/og/kubernetes-psp-deprecated.png
+lang: fr
+---
+
+Après l'annonce il y a quelques semaines de [la fin du support de Docker au
+sein de Kubernetes](https://particule.io/blog/kubernetes-docker-support/),
+c'est au tour des PodSecurityPolicy (PSP) de subir le même sort.
+
+Suite à [la PullRequest
+\#97171](https://github.com/kubernetes/kubernetes/pull/97171), les
+PodSecurityPolicy seront dépréciées à partir de la version `1.21` puis
+supprimées lors de la version `1.26`. Ce qui situe leur suppression dans 6
+releases, soit approximativement 18 mois au rythme actuel des releases.
+
+Nous vous avions déjà parlé des PSP dans [un précédent
+article](https://particule.io/blog/kubernetes-psp/). Ces ressources permettent
+d'enforcer certaines règles au niveau du pod, on peut citer par exemple :
+
+- impossibilité d'effectuer une élévation de privilèges (sudo)
+- impossibilité de démarrer en root
+- impossibilité d'utiliser des volumes hostPath
+- impossibilité de créer un pod "privileged"
+- etc
+
+Les PSP étaient activées au moyen d'un AdmissionControler au sein de l'API et
+les PSP elles mêmes devaient être utilisées par les utilisateurs créant le pod
+(ou le ServiceAccount) pour être prises en compte.
+
+Ce qu'on se rend tout de suite compte c'est que leur utilité n'est pas
+surcotée, les PSP sont des briques essentielles pour assurer une protection
+entre un pod et son host.
+
+![psp](https://rancher.com/img/blog/2020/pod-security/picture2.png)
+*© crédits Rancher*
+
+## Mais pourquoi les déprécier alors ?
+
+Il y a plusieurs raisons qui ont poussé la communauté à faire ce choix.
+
+Tout d'abord, où en sont les PSP aujourd'hui ? Et bien elles sont en beta.
```

Comment:
I'm not a 100% sure about this one, but I think it might be `Eh bien,` instead of `Et bien `